package.json

```diff
@@ -24,9 +24,9 @@
     "db:rollback": "node -r dotenv/config node_modules/knex/bin/cli.js migrate:rollback --knexfile src/db/knexfile.js",
     "test": "NODE_OPTIONS=--loader=testdouble c8 ava",
     "e2e": "playwright test src/e2e/",
-    "lint": "npm run lint:css && npm run lint:js",
+    "lint": "npm run lint:css && npm run lint:js && npm run lint:ts",
```

Comment:
Added `tsc` linter here, but I don't think the JS and TS linters are currently linting until we fix all the stylelint errors.
Not sure if we change this to use `||` instead of `&&` so it runs all 3 tasks (although it currently generates 450+ lines of output).

```diff
-"lint": "npm run lint:css && npm run lint:js && npm run lint:ts",
+"lint": "npm run lint:css || npm run lint:js || npm run lint:ts",
```